src/css/our-lessons.css

```diff
@@ -0,0 +1,109 @@
+.lessons-list {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.lesson-item {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 40px 20px;
+  border: 1px solid var(--green);
+  border-radius: 16px;
+}
+
+.lesson-item-title {
+  font-size: 20px;
+  font-weight: 700;
+  line-height: 1.2;
+  margin-bottom: 16px;
+}
+
+.lesson-item-price {
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 1.2;
```

Comment:
according to figma:
mobile - line-height: 36px; /* 112.5% */
tablet and desktop - line-height: 44px; /* 137.5% */

Reply:
Done